_posts/2023-06-11-vanilla-os-orchid-devlog.md

```diff
@@ -0,0 +1,195 @@
+
+The new version is not just an update; the entire underlying structure has been rethought and rewritten from scratch to be the ideal solution for all use cases due to the use of OCI images.
+
+In ABRoot v1, obtaining and applying system updates was done by leveraging the package manager in an isolated and atomic environment. In ABRoot v2, the update operation now utilizes OCI images, which are exact copies of the new system, downloaded and allocated using data deduplication and applied in a single atomic transaction. This way, all installations of Vanilla OS receive the exact copy of the update, making the system 100% reproducible and faithful to our tests.
+
+It automatically raises the question, "How do I install extra drivers?" in Vanilla OS Orchid, installation of the most common drivers is a graphical and effortless process. However, there are cases where it is necessary to install additional packages in the system. For this reason, we have introduced a package manager in ABRoot v2 that allows users to manage their packages using atomic transactions.
+
+Below is an example of the commands to add and remove packages:
+
+```bash
+abroot pkg add <your-package>
+abroot pkg remove <your-package>
+```
+
+Once a package is added or removed, it will not be installed or removed from the system immediately. It will be up to the user to apply them with the command:
+
+```bash
+abroot pkg apply
+```
+
+It will create a local image of the latest Vanilla OS version, adding or removing the desired packages. This new local image would be kept up to date automatically with the latest versions of Vanilla OS.
+
+### Always Bootable
+
+One of the main features of this version, is the change in the boot management. In v1, a single boot partition managed and contained the kernel images, initrd, and GRUB configuration. In v2, this process changes significantly as the boot partition gets divided into two parts: root boot and master boot.
+
+Each root partition has its own boot, containing the GRUB configuration, kernel, and initrd images. These configurations get dynamically managed by the master boot, a general GRUB configuration that handles switching and loading the boot from the correct root based on the update status.
+
+Thanks to this new structure, the update process never modifies the master boot except atomically, ensuring and preserving the system's boot.
+
+### Custom Images with Vib
+
+Since Vanilla OS is a general-purpose distribution, users may have specific needs that require modifications to the base structure of Vanilla OS, such as a different desktop environment. Thanks to the OCI image system, you can create your own custom image in just a few minutes by adding your customizations to the official image's recipe.
```

Comment:
```suggestion
Since Vanilla OS is a general-purpose distribution, users may have specific needs that require modifications to the base structure of Vanilla OS, such as a different desktop environment. Thanks to the OCI image system, you can create your own custom image in just a few minutes by adding your customizations on top of the official image's recipe.
```